Python/compile.c

```diff
@@ -3622,8 +3622,10 @@ maybe_optimize_method_call(struct compiler *c, expr_ty e)
 static int
 compiler_call(struct compiler *c, expr_ty e)
 {
-    if (maybe_optimize_method_call(c, e) > 0)
-        return 1;
+    int ret = maybe_optimize_method_call(c, e);
+    if (ret >= 0) {
```

Comment:
Why the change from `> 0` to `>= 0`?

Reply:
It is not obvious, but `maybe_optimize_method_call()` can return 0, which means that an exception was set. `compiler_call()` should return 0 in this case.